### ISSUE TITLE
Payment Information Review from Authenticated Experience

### ISSUE THREAD

**Issue by tressaellen** [department-of-veterans-affairs/va.gov-team#34015]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [ ] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [ ] Making revisions to an existing online form, app, or tool on VA.gov
- [x] Something else (please add details in the next question)

## What do you need help with?
  - [x] Content support
  - [x] IA support

We're launching a payment information section on My VA and need some help with the content. 

  
## Supporting Artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/payment-history#project-outline-add-paymentfinancial-info-to-the-my-va-dashboard)
- Link to or attach user flows:
- Link to prototype or mockups: [Mural Board - Round 4 Final](https://app.mural.co/t/vsa8243/m/vsa8243/1640028268317/2ba4e1b2846c87f5d0bfc6bcb643bef84720bea1?sender=tressafurner7646)
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: TBD
- Estimated staging review date: TBD
- Content and IA work needed by: End of sprint 63

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [x] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: Authenticated Experience
-	OCTO-DE product owner: Samara Strauss
-	Product manager: Heather Justice
-	Designer: Tressa Furner and Liz Lantz
-	FE engineer: Taylor Mitchell
-	Product/team Slack channel: #vsa-authd-exp

**Comment by hdjustice** [comment]
@mnorthuis We opened this Content & IA ticket this week and Tressa posted in the [Sitewide Content slack](https://dsva.slack.com/archives/C01K37HRUAH/p1639417624186500) on Monday.

**Comment by mnorthuis** [comment]
Thank you @hdjustice!  I totally missed this!   Adding IA to the request for work around the content hierarchy and groupings on the My VA page related to financial "things".

**Comment by andaleliz** [comment]
@mnorthuis, @DanielleThierryUSDSVA, and/or @RLHecht I'm taking the lead on this work for Tressa. I think the booster got both Tressa and Danielle last week, so not sure any progress has been made on this.  Tressa was right in her slack post that we are hoping to get this wrapped up in this sprint.  I'll bring this to office hours Thursday but welcome any thoughts ahead of time if you have a chance to look.

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @andaleliz. That sounds great to bring this to health office hours. I'll see if @mnorthuis can potentially make it as well and we can talk through. Thanks!

**Comment by andaleliz** [comment]
We discussed this in office hours and decided on the following changes:

- Update **Payment information** heading to **Benefit payments and debts**
- Update the alert copy from a single link to improve clarity about where the user would go if they clicked the link, and to align with other similar CTAs on VA.gov. The new copy is: You have 3 outstanding debt. [Manage your VA debt]
- Update the card copy to use active voice. It now reads: We deposited [$$] in your account ending in [acct] on [date].

Closing this out - thanks for your help Danielle and Mikki!
